lib/rules/strict.js

```diff
@@ -17,7 +17,9 @@ var messages = {
     global: "Use the global form of \"use strict\".",
     multiple: "Multiple \"use strict\" directives.",
     never: "Strict mode is not permitted.",
-    unnecessary: "Unnecessary \"use strict\" directive."
+    unnecessary: "Unnecessary \"use strict\" directive.",
+    unnecessaryInModules: "Unnecessary \"use strict\" directive, because inside of modules.",
```

Comment:
Why do these messages need to be different from the regular "unnecessary" message? In a related note, the "global form" vs. the "function form" of a directive don't really mean anything to most users (including me).